src/google/protobuf/stubs/time.cc

```diff
@@ -1,6 +1,10 @@
 #include <google/protobuf/stubs/time.h>
 
+#if __cplusplus > 199711L
```

Comment:
I don't think there is any need for this conditional (and the one below) because starting from version 3.5.0 we require C++11 or newer.

Reply:
Removed.